Makefile

```diff
@@ -60,7 +60,6 @@ ifndef REGISTRY_MANIFEST_ID
 	@echo "Error! REGISTRY_MANIFEST_ID must be set."; exit 1
 endif
 	gcloud container clusters get-credentials registry-backend --zone us-central1-a
-	export DEFAULT_SA_KEY=$(kubectl get secrets -o jsonpath="{.items[0].metadata.name}" | grep default-token)
```

Comment:
Why was this removed? It doesn't seem related to testing.

Reply:
The default SA key is not required in the job since the registry server we are deploying in GKE is without an auth server. The key was present previously, but was not made use of.

I made changes to `deployments/controller/gke-job/cron-job.yaml` as well to take out the `APG_REGISTRY_TOKEN` env variable.

Reply:
SGTM - thanks!